Source/RxSwift/zipWith.swift

```diff
@@ -0,0 +1,24 @@
+//
+// Created by Arjan Duijzer on 26/12/2017.
+// Copyright (c) 2017 RxSwiftCommunity. All rights reserved.
+//
+
+import RxSwift
+
+extension ObservableConvertibleType {
+
+    /**
+     Merges the specified observable sequences into one observable sequence by using the selector function whenever all of the observable sequences have produced an element at a corresponding index.
+
+     - seealso: [zip operator on reactivex.io](http://reactivex.io/documentation/operators/zip.html)
+
+     - Parameters:
+        - second: A second Observable<T> to zip with `self`
```

Comment:
I think the name of the second parameter is actually `with`, since `second` is the internal name.
I would also argue it's probably better to just call it `zip(with observable:)`

Reply:
updated